Python/Directory-Tree-Generator/dstree/dstree.py

```diff
@@ -0,0 +1,58 @@
+import os
+import pathlib
+
+PIPE = "|"

```

Comment:
There is a unicode character which would be better suitable instead of a simple `|`, namely `U+2502`